tools/travis_script.py

```diff
@@ -46,44 +52,107 @@ def run_docker():
 
 def exec_docker(cwd, cmd):
     exec_cmd = 'cd %s && ' % cwd + ' '.join(cmd)
-    ex.check_run_cmd('docker', ['exec', '-it', DOCKER_NAME,
-                                'bash', '-c', exec_cmd])
+    ex.check_run_cmd('docker', [
+                     'exec', '-it', DOCKER_NAME, 'bash', '-c', exec_cmd])
 
 def set_release_config_tizenrt():
-    exec_docker(DOCKER_ROOT_PATH, ['cp', 'tizenrt_release_config',
-                                   fs.join(DOCKER_TIZENRT_OS_PATH, '.config')])
+    exec_docker(DOCKER_ROOT_PATH, [
+                'cp', 'tizenrt_release_config',
+                fs.join(DOCKER_TIZENRT_OS_PATH, '.config')])
+
+def build_iotjs(buildtype, args=[]):
+    exec_docker(DOCKER_IOTJS_PATH, [
+                './tools/build.py',
+                '--clean',
+                '--buildtype=' + buildtype] + args)
 
 if __name__ == '__main__':
-    run_docker()
+    if os.getenv('RUN_DOCKER') == 'yes':
+        run_docker()
 
-    test = os.environ['OPTS']
+    test = os.getenv('OPTS')
```

Comment:
Why aren't we using cmdline arguments instead? IMHO this is a bad practice.

Reply:
In `.travis.yml`, environment variables have been used when running scripts.

Before code,
```
`tools/precommit.py $OPTS;
```

Travis branches test with environment variables.
```
env:
   matrix:
```
If we use cmdline arguments in python script, we need to add code that handle arguments. But it also use environment variables. Do you want to put environment variables as arguments in python script as before?

Reply:
Yes, I think it is better, but it is only my personal preference.

Reply:
IMHO, this script is named for travis test. So I can accept environment dependencies from travis. I assume that this script will never be locally used like how `precommit.py` was wrongly used.

Reply:
@daeyeon in that case we can keep it.